deploy/role_binding.yaml

```diff
@@ -6,7 +6,7 @@ subjects:
 - kind: ServiceAccount
   name: openshift-pipelines-operator
   # Replace this with the namespace the operator is deployed in.
-  namespace: tekton-pipelines
+  namespace: dev-openshift-pipelines-operator
```

Comment:
why has the namespace changed?

Reply:
Openshift will no let you name a namespace with a string starting with 'openshift'

Reply:
we need a namespace to run the tests. The operator should have necessary permissions in that namespace. so i set the namespace as `dev-openshift-pipelines-operator` in the role_binding and used it in the run test command
```
operator-sdk test local ./test/e2e --namespace dev-openshift-pipelines-operator --debug
```

Reply:
while using the operator with OLM the permissions are set from CSV not from role_binding.yaml

Openshift will no let you name a namespace with a string starting with 'openshift'. Hence the `dev-` prefix